playground/App.vue

```diff
@@ -1,105 +1,37 @@
 <template>
   <div id="app">
-    <ResizablePanels>
-      <FormRenameStep
-        v-if="isEditingStep"
-        :initialValue="initialValue"
-        slot="left-panel"
-        @cancel="cancelStepEdition"
-        @formSaved="saveStep"
-      />
-      <Pipeline v-else slot="left-panel"/>
-      <DataViewer slot="right-panel" @stepCreated="openStepForm"/>
-    </ResizablePanels>
+    <vqb/>
     <pre><code>{{ code }}</code></pre>
   </div>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import Vue from 'vue';
+import { Component } from 'vue-property-decorator';
 import { Getter, Mutation, State } from 'vuex-class';
-import { VQBState, activePipeline } from '@/store/state';
-import { Pipeline, PipelineStep } from '@/lib/steps';
-import { DataSet } from '@/lib/dataset';
-import { MongoResults, mongoResultsToDataset } from '@/lib/dataset/mongo';
-import {
-  DataViewer,
-  FormRenameStep,
-  Pipeline as PipelineComponent,
-  ResizablePanels,
-  getTranslator,
-} from '../dist/vue-query-builder.common.js';
-import _ from 'lodash';
+import { VQBState } from '@/store/state';
+import { Pipeline } from '@/lib/steps';
+import { Vqb, getTranslator } from '../dist/vue-query-builder.common.js';
 
 const mongo36translator = getTranslator('mongo36');
 
 @Component({
   components: {
-    DataViewer,
-    FormRenameStep,
-    Pipeline: PipelineComponent,
-    ResizablePanels,
+    Vqb,
   },
+
 })
 export default class App extends Vue {
   @State pipeline!: Pipeline;
-  @State domains!: Array<string>;
 
   @Getter activePipeline!: Pipeline;
```

Comment:
@adimasci Is not `activePipeline` the only piece of data that we need to get from the store in this file as such ?
`pipeline`, and `setDomains` and `setDataset` mutations are not directly used here right ?